assets/js/blocks/collection-filters/inner-blocks/stock-filter/edit.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronDown } from '@wordpress/icons';
+import { ProductQueryContext as Context } from '@woocommerce/blocks/product-query/types';
+import { CheckboxList } from '@woocommerce/blocks-components';
+import Label from '@woocommerce/base-components/filter-element-label';
+import FormTokenField from '@woocommerce/base-components/form-token-field';
+import type { BlockEditProps } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { BlockProps } from './types';
+import { Inspector } from './inspector';
```

Comment:
Just sharing what I'm doing with Price and Attribute filter: I put child component of the Edit into a folder called `components`, so the block folder only has the basic and common files. I found that quite clean, but I know that every one may have different preferences.

Reply:
I am happy to stick to a consistent convention 😄 will fix.

Reply:
Fixed [here](https://github.com/woocommerce/woocommerce-blocks/pull/11663/commits/b9c0fe3af8603cc7eb53638eb3d237119be8ac85)